src/Sentry.Unity/SentryScene.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+
+namespace Sentry.Unity
+{
+    public class SentryScene : MonoBehaviour
```

Comment:
Adding a callback to a button relies on a MonoBehaviour being a permanent part of the scene. `SentryMonoBehaviour` get created during runtime and the callback would fail.